README.md

```diff
@@ -41,6 +41,9 @@
 </div>
 <hr>
 
+[![GitHub Tread](https://trendshift.io/api/badge/repositories/8648)](https://trendshift.io/repositories/8648)
```

Comment:
Amazing! Can we put this above the `<hr>` and center it!

Reply:
Done. Move it to the top.